umh.docs.umh.app/content/en/docs/datacontracts/_index.md

```diff
@@ -0,0 +1,206 @@
+timestamp in the message, you know that you could look them up in the
+Management Console.
+
+These rules ensure that the data can be written into the intended database
+tables without causing errors, and that the data can be read by other programs,
+as it is known what data and structure to expect.
+
+For example, the timestamp is an easy way to avoid errors by making each message
+idempotent (can be safely processed multiple times without changing the result).
+Each message in a tag is made completely unique by its timestamp, which is
+critical because messages are sent using "at least once" semantics, which can
+lead to duplicates. With idempotency, duplicate messages are ignored, ensuring
+that each message is only stored once in the database.
+
+{{% notice note %}}
+If you want a lot more information and really dive into the reasons for this
+approach, we recommend our article about
+[Data Modeling in the UNS](https://learn.umh.app/lesson/data-modeling-in-the-unified-namespace-mqtt-kafka/)
+on our Learn page.
+<!-- Add link to Jeremys upcoming article, once it is released. -->
+{{% /notice %}}
+
+## Rules of a Data Contract
+
+Data Contracts can enforce a number of rules. This section provides an overview
+of the two rules that are enforced by default. The specifics can vary between
+Data Contracts; therefore, detailed information about the
+[Historian Data Contract](https://umh.docs.umh.app/docs/datacontracts/historian/)
+and [Custom Data Contracts](https://umh.docs.umh.app/docs/datacontracts/customdatacontracts/)
+is provided on their respective pages.
+
+### Topic Structure
+
+As mentioned in the example, messages in the UMH must follow our ISA-95
+compliant structure in order to be processed. The structure itself can be
+divided into several sections.
+
+{{<mermaid theme="neutral" >}}
+flowchart LR
+    umh --> v1
+    v1 --> enterprise
+    enterprise -->|Optional| site
+    site -->|Optional| area
+    area -->|Optional| productionLine
+    productionLine -->|Optional| workCell
+    workCell -->|Optional| originID
+    originID -.-> _schema["_schema (Ex: _historian, _custom)"]
+    _schema -->_opt["Schema dependent context"]
+
+    classDef mqtt fill:#00dd00,stroke:#333,stroke-width:4px;
+    class umh,v1,enterprise,_schema mqtt;
+    classDef optional fill:#77aa77,stroke:#333,stroke-width:4px;
+    class site,area,productionLine,workCell,originID optional;
+    
+    enterprise -.-> _schema
+    site -.-> _schema
+    area -.-> _schema
+    productionLine -.-> _schema
+    workCell -.-> _schema
+
+{{</ mermaid >}}
+
+You can check if your topics are correct in the validator below.
+
+{{% topic_decoder %}}
+
+#### Prefix
+
+The first section is the mandatory prefix: `umh.v1.` It ensures that the
+structure can evolve over time without causing confusion or compatibility
+problems.
+
+#### Location
+
+The next section is the Location, which consists of six parts:
+`enterprise.site.area.productionLine.workCell.originID`.
```

Comment:
FYI: in the future we will Call this levels instead https://linear.app/united-manufacturing-hub/issue/ENG-1731/make-isa-95-hierarchy-levels-generic-and-configurable-allowing